sharding-core/sharding-core-common/src/main/java/org/apache/shardingsphere/core/metadata/column/EncryptColumnMetaData.java

```diff
@@ -37,9 +37,9 @@
     
     private final String assistedQueryColumnName;
     
-    public EncryptColumnMetaData(final String name, final String dataType, final boolean primaryKey, 
+    public EncryptColumnMetaData(final String name, final String dataType, final boolean primaryKey, final boolean nullable, final boolean autoIncrement,
```

Comment:
Why change the parameter to 'nullable' which meaning is opposite to 'notNull'. If 'notNull' is true, then 'nullable' will be false.

Reply:
I hava already fixed it. Unified name is `notNull`.

Reply:
I have refreshed the codes, but still can see nullable.
![image](https://user-images.githubusercontent.com/24643893/67738490-e26f1c80-fa49-11e9-95ce-e9cc148979a9.png)